pkg/controller/statusmanager/pod_status.go

```diff
@@ -86,7 +86,11 @@ func (status *StatusManager) SetFromPods() {
 			progressing = append(progressing, fmt.Sprintf("DaemonSet %q update is rolling out (%d out of %d updated)", dsName.String(), ds.Status.UpdatedNumberScheduled, ds.Status.DesiredNumberScheduled))
 			dsProgressing = true
 		} else if ds.Status.NumberUnavailable > 0 {
-			progressing = append(progressing, fmt.Sprintf("DaemonSet %q is not available (awaiting %d nodes)", dsName.String(), ds.Status.NumberUnavailable))
+			if strings.Contains(dsName.String(), "multus-admission-controller") {
```

Comment:
Is there a better way to store this string as a constant, or pull it from another place? I couldn't quite find the right place to store it.

Reply:
We already use the "non-critical" annotation to identify the multus admission controller, so just do `if isNonCritical(ds) {`.